drivers/gpio/gpio_andes_atcgpio100.c

```diff
@@ -140,7 +141,7 @@ static int gpio_atcgpio100_config(const struct device *port,
 		key = k_spin_lock(&data->lock);
 
 		/* Set de-bounce */
-		if (flags & GPIO_INT_DEBOUNCE) {
+		if (flags & ATCGPIO100_GPIO_INT_DEBOUNCE) {
```

Comment:
I think we should get rid of this misleading `INT_` component from the names of debounce flags. Unlike all other `GPIO_INT_*` flags, the debounce flag is supposed to be used in calls to `gpio_pin_configure()`, not `gpio_pin_interrupt_configure()`.

Reply:
@anangl Thank you for reviewing. I have removed the `_INT` part of the name for the SoC specific debounce flags.